frontend/src/hooks/useToast.tsx

```diff
@@ -14,8 +14,7 @@ const useToast = () => {
     const setToastApiError = useCallback(
         (text: string, overrides?: IToast) => {
             setToast({
-                title: 'Something went wrong',
-                text,
+                title: text,
```

Comment:
This may be the most controversial change here. We can stick with just "something went wrong", of course, but that'd require also updating a bunch of other files. Besides, that text isn't very helpful. On the other hand, the regular error text can get pretty wordy 🤷🏼